### PR TITLE
Add HighEntropyVA to building pwsh

### PR DESCRIPTION
**Pull request by TravisEz13**

### CODE REVIEW

**Review by SteveL-MSFT — APPROVED** [review]

File: src/powershell-unix/powershell-unix.csproj [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <AssemblyName>pwsh</AssemblyName>
     <OutputType>Exe</OutputType>
     <RuntimeIdentifiers>linux-x64;osx.10.12-x64;</RuntimeIdentifiers>
+    <HighEntropyVA>true</HighEntropyVA>
```
Comment (iSazonov) [inline src/powershell-unix/powershell-unix.csproj]: Why we don't add this in all csproj files via `PowerShell.Common.props`?

**Review by iSazonov — APPROVED** [review]
LGTM.

One minor question. Sometimes msbuild is a surprise. How we can check that the option really works?

**Comment by TravisEz13** [comment]
We will run binskim during the official build which will tell us.
https://github.com/Microsoft/binskim

**Comment by daxian-dbw** [comment]
We don't build `pwsh.exe`, so is the executable also has this option set on?
According to https://msdn.microsoft.com/en-us/library/jj835761.aspx,  If not, powershell may not be able to benefit from this option.

**Comment by iSazonov** [comment]
I see the option in CoreCLR https://github.com/dotnet/coreclr/search?utf8=%E2%9C%93&q=HIGHENTROPYVA&type=

**Review by daxian-dbw — APPROVED** [review]
LGTM. :shipit:

**Pull request merged**